XamarinCommunityToolkit/Views/SegmentedView/Android/FormsSegments.android.cs

```diff
@@ -0,0 +1,294 @@
+			{
+				mode = value;
+				InitializeSegments();
+			}
+		}
+
+		CornerRadius cornerRadius = 8.0f;
+
+		public CornerRadius CornerRadius
+		{
+			get => cornerRadius;
+			set
+			{
+				cornerRadius = value;
+				InitializeSegments();
+			}
+		}
+
+		AColor tintColor = AColor.Rgb(14, 98, 255);
+
+		public AColor TintColor
+		{
+			get => tintColor;
+			set
+			{
+				tintColor = value;
+				InitializeSegments();
+			}
+		}
+
+		AColor backgroundColor = AColor.Transparent;
+
+		public AColor BackgroundColor
+		{
+			get => backgroundColor;
+			set
+			{
+				backgroundColor = value;
+				SetBackgroundColor(value);
+			}
+		}
+
+		public ObservableCollection<string> Children { get; } = new ObservableCollection<string>();
+
+		RadioButton currentSegment;
+
+		public RadioButton CurrentSegment
+		{
+			get => currentSegment;
+			set
+			{
+				currentSegment = value;
+			}
+		}
+
+		public event EventHandler<SelectedPositionChangedEventArgs> SegmentSelected;
+
+		int strokeWidth;
+		bool disposed;
+
+		public FormsSegments(Context context)
+            : base(context)
+		{
+			this.context = context;
+			Children.CollectionChanged += CollectionChanged;
+			Build();
+		}
+
+		void CollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
+		{
+			InitializeSegments();
+		}
+
+		void Build()
+		{
+			strokeWidth = (int)context.ToPixels(defaultStrokeWidth);
+
+			SetBackgroundColor(backgroundColor);
+
+			Orientation = Orientation.Horizontal;
+			LayoutParameters = new LayoutParams(LayoutParams.MatchParent, LayoutParams.WrapContent);
+			CheckedChange += OnCheckChanged;
+		}
+
+		public void InitializeSegments()
+		{
+			if (ChildCount > 0)
+				RemoveAllViews();
+
+			for (var i = 0; i < Children.Count; i++)
+			{
+				var rb = InsertSegment(Children[i].ToString(), i);
+				if (i == 0)
+					rb.Checked = true;
+			}
+		}
+
+		RadioButton InsertSegment(string title, int index)
+		{
+			var isLeft = index == 0;
+			var position = isLeft ? Position.Left : index == Children.Count - 1 ? Position.Right : Position.Middle;
+			var rb = GetRadioButton(title, position);
+			AddView(rb);
+			return rb;
+		}
+
+		AContentRes.ColorStateList TextColorSelector => new AContentRes.ColorStateList(
+			new int[][] // states
+			{
+				new int[] { ARes.Attribute.StateChecked },
+				new int[] { -ARes.Attribute.StateChecked }
+			},
+			new int[] // colors
+			{
+				AColor.White, TintColor
+			});
+
+		AContentRes.ColorStateList SegmentColorSelector => new AContentRes.ColorStateList(
+			new int[][] // states
+			{
+				new int[] { ARes.Attribute.StateChecked },
+				new int[] { -ARes.Attribute.StateChecked }
+			},
+			new int[] // colors
+			{
+				TintColor, BackgroundColor
+			});
+
+		void OnCheckChanged(object sender, CheckedChangeEventArgs e)
+		{
+			CurrentSegment = FindViewById<RadioButton>(e.CheckedId);
+			SegmentSelected?.Invoke(this, new SelectedPositionChangedEventArgs(IndexOfChild(CurrentSegment)));
+		}
+
+		#region Drawable Resources
```

Comment:
Can you remove this `#region` and others that may have in this PR?

Reply:
Done.